client/src/Components/Login.js

```diff
@@ -132,6 +137,16 @@ const Login = () => {
             Continue with Google
           </h4>
         </div>
+        <div className={classes.loginService}>
+          <FacebookLogin
+            appId="534803271003174"
```

Comment:
Is this specific to your account? If yes, let's make this config based in the .env file